### PR TITLE
Version 7.7.6

### PR DESCRIPTION
**Pull request by hobinjk**
Updates Chrome and Firefox versions, pointing FirefoxUpdate.rdf to the new release. Should be merged after #1291

### CODE REVIEW

File: Extensions/dist/page/FirefoxUpdate.rdf [hunk 1]
```diff
@@ -12,17 +12,17 @@
...
           <RDF:Description>
-            <em:version>7.7.5-mp</em:version> <!-- This is the version number of the add-on -->
+            <em:version>7.7.6</em:version> <!-- This is the version number of the add-on -->
 
             <!-- One targetApplication for each application the add-on is compatible with -->
             <em:targetApplication>
               <RDF:Description>
                 <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
-                <em:minVersion>35.0</em:minVersion>
+                <em:minVersion>51.0a1</em:minVersion>
```
Comment (nightpool) [inline Extensions/dist/page/FirefoxUpdate.rdf]: this is required for the embedded webextension?

File: Extensions/dist/page/framework_version.json [hunk 2]
```diff
@@ -1 +1 @@
-{"server":"up","frameworks":[{"name":"firefox","version":"7.7.1"},{"name":"chrome","version":"7.7.1"}]}
+{"server":"up","frameworks":[{"name":"firefox","version":"7.7.6"},{"name":"chrome","version":"7.7.6"}]}
```
Comment (nightpool) [inline Extensions/dist/page/framework_version.json]: oh, yeah. that.

File: Extensions/dist/page/FirefoxUpdate.rdf (hunk 1 above)
Comment (hobinjk) [inline Extensions/dist/page/FirefoxUpdate.rdf]: Yes, the embedded webextension feature was added in Firefox 51

**Comment by hobinjk** [comment]
This is ready to go! I published a release at https://github.com/new-xkit/XKit/releases/tag/firefox-v7.7.6 and have a blog post drafted. Feedback are welcome on both

**Pull request merged**